var/spack/repos/builtin/packages/camp/package.py

```diff
@@ -10,10 +10,8 @@
 
 def hip_repair_options(options, spec):
     # there is only one dir like this, but the version component is unknown
-    options.append(
-        "-DHIP_CLANG_INCLUDE_PATH="
-        + glob.glob("{}/lib/clang/*/include".format(spec["llvm-amdgpu"].prefix))[0]
-    )
+    path = glob.glob("{}/lib/clang/*/include".format(spec["llvm-amdgpu"].prefix))[0]
+    options.append(cmake_cache_path("HIP_CLANG_INCLUDE_PATH", path))
```

Comment:
We can remove this change - was necessary in debugging, but should be fixed in other PRs